build error on master

```../../IBAMR/lib/../src/IB/IBExplicitHierarchyIntegrator.cpp: In member function 'virtual std::pair<std::vector<Eigen::Matrix<double, 2, 1>, Eigen::aligned_allocator<Eigen::Matrix<double, 2, 1> > >, std::vector<Eigen::Matrix<double, 2, 1>, Eigen::aligned_allocator<Eigen::Matrix<double, 2, 1> > > > IBAMR::IBExplicitHierarchyIntegrator::collectAllMarkers() const':
../../IBAMR/lib/../src/IB/IBExplicitHierarchyIntegrator.cpp:523:17: error: could not convert '<brace-enclosed initializer list>()' from '<brace-enclosed initializer list>' to 'std::pair<std::vector<Eigen::Matrix<double, 2, 1>, Eigen::aligned_allocator<Eigen::Matrix<double, 2, 1> > >, std::vector<Eigen::Matrix<double, 2, 1>, Eigen::aligned_allocator<Eigen::Matrix<double, 2, 1> > > >'
  523 |         return {};
      |                 ^
      |                 |
      |                 <brace-enclosed initializer list>

--- a/src/IB/IBExplicitHierarchyIntegrator.cpp
+++ b/src/IB/IBExplicitHierarchyIntegrator.cpp
@@ -525,7 +525,12 @@ IBExplicitHierarchyIntegrator::collectAllMarkers() const
     if (d_markers)
         return d_markers->collectAllMarkers();
     else
-        return {};
+    {
+        // Some older compilers cannot convert just '{}' into a std::pair so help it out
+        EigenAlignedVector<IBTK::Point> positions;
+        EigenAlignedVector<IBTK::Vector> velocities;
+        return std::make_pair(positions, velocities);
+    }
 } // collectAllMarkers
 
 void

